uni/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -30,6 +31,7 @@ class AppSharedPreferences {
     FavoriteWidgetType.busStops
   ];
   static const String filteredExamsTypes = 'filtered_exam_types';
+  static const String hiddenExams = 'hidden_exams';
```

Comment:
Would look better to have all the keys located in the same location.
Can you move 'hiddenExams', 'filteredExamsTypes' and 'favorite_cards' up, below 'themeMode'?
I know it's out of scope of this issue, but we can make this little change right now.